examples/ristretto/src/ristretto.rs

```diff
@@ -111,13 +117,50 @@ fn D_MINUS_ONE_SQ() -> FieldElement {
     ))
 }
 
+// === Special points === //
+
+pub fn BASE_POINT_ENCODED() -> RistrettoPointEncoded {
+    RistrettoPointEncoded::from_seq(&byte_seq!(
+        0xe2u8, 0xf2u8, 0xaeu8, 0x0au8, 0x6au8, 0xbcu8, 0x4eu8, 0x71u8, 0xa8u8, 0x84u8, 0xa9u8,
+        0x61u8, 0xc5u8, 0x00u8, 0x51u8, 0x5fu8, 0x58u8, 0xe3u8, 0x0bu8, 0x6au8, 0xa5u8, 0x82u8,
+        0xddu8, 0x8du8, 0xb6u8, 0xa6u8, 0x59u8, 0x45u8, 0xe0u8, 0x8du8, 0x2du8, 0x76u8
+    ))
+}
+
+pub fn BASE_POINT() -> RistrettoPoint {
+    decode(BASE_POINT_ENCODED()).unwrap()
```

Comment:
Would it make sense to have a decoded version as const to avoid the `unwrap()`?

Reply:
Could be done, but why is the unwrap here something we would want to avoid?

Reply:
It's just a potential panic that is unnecessary. But not a big deal here.